nifi-nar-bundles/pom.xml

```diff
@@ -114,6 +114,7 @@
         <module>nifi-py4j-bundle</module>
         <module>nifi-compress-bundle</module>
         <module>nifi-opentelemetry-bundle</module>
+        <module>nifi-apicurio-bundle</module>
```

Comment:
The `nar` also needs adding to the `nifi-assembly` if going to be included in the build